main.js

```diff
@@ -23,30 +28,44 @@ const healthChecks = require('./src/lib/health-checks');
 const instrumentListen = require('./src/lib/instrument-listen');
 const guessAppDetails = require('./src/lib/guess-app-details');
 
-const cache = require('./src/middleware/cache');
+const { cache } = require('./src/middleware/cache');
 const robots = require('./src/middleware/robots');
 const security = require('./src/middleware/security');
 const vary = require('./src/middleware/vary');
 const logVary = require('./src/middleware/log-vary');
 const anon = require('./src/middleware/anon');
-const teapot = fs.readFileSync(path.join(__dirname, 'src/assets/teapot.ascii'), 'utf8');
-
-const getAppContainer = options => {
-
-	options = Object.assign({}, {
-		withBackendAuthentication: true,
-		withFlags: false,
-		withAb: false,
-		withConsent: false,
-		withServiceMetrics: true,
-		healthChecks: []
-	}, options || {});
+const teapot = fs.readFileSync(
+	path.join(__dirname, 'src/assets/teapot.ascii'),
+	'utf8'
+);
+
+/**
+ * @param {AppOptions} options
+ * @returns {import("./typings/n-express").AppContainer}
```

Comment:
**nitpick**: for consistency, and maybe it'll read cleaner, it would be good to have these imports at the top of the file with the others, not inline

Reply:
My logic was to put the imports inline if the type was only used once, and at the top if it was reused, but consistency probably is more readable!

Reply:
ah! that probably does make sense tbh. up to you!